docs/using.md

```diff
@@ -0,0 +1,282 @@
+# Using Quix Streams
+
+In this topic you will learn how to use Quix Streams to perform two types of data processing:
+
+1. **One message at a time processing** - Here the message received contains all required data for processing. No state needs to be preserved between messages, or between replicas. The data from the message is used to calculate a new value, which is then typically published to the output stream.
```

Comment:
Stateless processing is a more frequent term. I would use that in bold and then mention "One message at a time processing" in the description.

Reply:
Fixed.